docs/pages/includes/config-reference/app-service.yaml

```diff
@@ -5,6 +5,10 @@ app_service:
     # Application Service is working correctly. The app outputs JWTs so it can
     # be useful when extending your application.
     debug_app: true
+    # Enables the builtin Teleport demo MCP server that shows current user and
+    # session information. Wildcard must be allowed for "app_labels" by your
+    # role to access this MCP server.
```

Comment:
Do we really check `*` for app labels and not any particular `mcp.tools`?

Is there any harm in allowing this for everyone?

Reply:
i don't think there is harm to allow everything but i don't want to make special app_labels matching logic for this either. Access role by default has wildcard so i don't think it will be a big issue.